open_webui_plugins_builder/tests/test_config.py

```
@@ -0,0 +1,274 @@
+    raw = {
+        "meta": {"schema_version": 1},
+        "settings": {"local_import_roots": ["owui_ext.shared"]},
+        "targets": [
+            {
+                "name": "invalid",
+                "source": "src/a.py",
+                "output": "../../tmp/out.py",
+            }
+        ],
+    }
+    with pytest.raises(BuildError, match="must not contain '..'"):
```

Ruff (0.15.12)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)